parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -13,6 +13,10 @@ K8S_DOWNLOADS_DIR="/opt/kubernetes/downloads"
 UBUNTU_RELEASE=$(lsb_release -r -s)
 TELEPORTD_PLUGIN_DOWNLOAD_DIR="/opt/teleportd/downloads"
 TELEPORTD_PLUGIN_BIN_DIR="/usr/local/bin"
+KRUSTLET_DOWNLOAD_DIR="/opt/KRUSTLET/downloads"
+KRUSTLET_BIN_DIR="/usr/local/bin"
+KRUSTLET_VERSION="v0.7.0"
+KRUSTLET_URL="https://krustlet.blob.core.windows.net/releases/krustlet-${KRUSTLET_VERSION}-linux-amd64.tar.gz"
```

Comment:
with our new binary build, this will be: https://acs-mirror.azureedge.net/krustlet/v0.7.0/linux/amd64/krustlet-wasi, and it won't be a tarball any more, just the binary. 

We need to download the binary, chmod +x it, and then move it into /usr/local/bin/krustlet-wasi

Reply:
see https://github.com/Azure/AgentBaker/blob/45e0c3bfd552eaaa0935e5f1b92168a46563dc9e/vhdbuilder/scripts/linux/tool_installs.sh#L12-L38 for a good example (only the bpftrace binary, there is other stuff in /tools you can ignore)

Reply:
I went ahead and took a stab at this so we don't have to wait for a fresh vhd build in the morning.

Reply:
we'll still need the prod binaries, this is the staging upstream storage account which is for testing only.

sercan will need to approve the prod build in the morning and then we can use the acs-mirror.azureedge.net CDN